vms/platformvm/utxo/handler.go

```diff
@@ -171,7 +171,7 @@ func (h *handler) Spend(
 	kc := secp256k1fx.NewKeychain(keys...) // Keychain consumes UTXOs and creates new ones
 
 	// Minimum time this transaction will be issued at
-	now := uint64(h.clk.Time().Unix())
+	now := h.clk.Time().Unix()
```

Comment:
```suggestion
	now := uint64(h.clk.Time().Unix())
```
I think we change this back to how it was done before and remove the noop diff.